NitroxInstaller/Product.wxs

```diff
@@ -100,6 +103,14 @@
       </Component>
     </ComponentGroup>
   </Fragment>
+  
+  <Fragment>
+    <ComponentGroup Id="SUBNAUTICAMONO_files" Directory="SUBNAUTICAMONO">
+      <Component Id="discordRPC.dll" Guid="*">
+        <File Id="discordRPC.dll" Name="discord-rpc.dll" Source="$(var.NitroxClient_TargetDir)MonoBehaviours/DiscordRP/discord-rpc.dll" />
```

Comment:
This doesn't seem like a good location for the DLL (path is along side .cs files). Also I prefer not to put .dll files on a git repo directly but if it has to happen then put the DLL in the resources. 

We should discuss the best solution on Discord.

Reply:
No this path is in the output dir. VS copies the file on compiling there. So the path is"..\Nitrox\bin\Debug\MonoBehaviours\DiscordRP".
But yeah I will move it to the resources dir.